source/shared/cpp/ObjectModel/BaseCardElement.cpp

```diff
@@ -14,11 +14,20 @@ BaseCardElement::BaseCardElement(
     m_spacing(spacing),
     m_separator(separator)
 {
+    BaseCardElement::populateKnownPropertiesSet();
 }
 
 BaseCardElement::BaseCardElement(CardElementType type) :
     m_type(type), m_spacing(Spacing::Default)
 {
+    BaseCardElement::populateKnownPropertiesSet();
+}
+
+void BaseCardElement::populateKnownPropertiesSet(void)
```

Comment:
>p [](start = 22, length = 1)

Uppercase the first letter of the function name please

Reply:
thanks

---
In reply to: [171354215](https://github.com/Microsoft/AdaptiveCards/pull/1179#discussion_r171354215) [](ancestors = 171354215)